code/controllers/subsystem/dcs.dm

```diff
@@ -7,10 +7,22 @@ PROCESSING_SUBSYSTEM_DEF(dcs)
 /datum/controller/subsystem/processing/dcs/Recover()
 	comp_lookup = SSdcs.comp_lookup
 
-/datum/controller/subsystem/processing/dcs/proc/GetElement(eletype)
-	. = elements_by_type[eletype]
+/datum/controller/subsystem/processing/dcs/proc/GetElement(datum/element/eletype, ...)
+	var/element_id = eletype
+	
+	if(initial(eletype.element_flags) & ELEMENT_BESPOKE)
+		var/list/fullid = list("[eletype]")
+		for(var/i in initial(eletype.id_arg_index) to length(args))
+			var/argument = args[i]
+			if(istext(argument) || isnum(argument))
+				fullid += "[argument]"
+			else
+				fullid += "[REF(argument)]"
```

Comment:
Add mention to docs about this using REF so it will be obvious when someone adds list(a,b,c) as argument

Reply:
I improved the comment on the define